dts/bindings/sensor/nxp,lpcmp.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP low-power analog comparator (LPCMP)
+
+compatible: "nxp,lpcmp"
+
+include: [sensor-device.yaml, pinctrl-device.yaml]
+
+properties:
+  interrupts:
+    required: true
+
+  reg:
+    required: true
+
+  enable-output-pin:
+    type: boolean
+    description: |
+      Decide whether to enable the comparator is available in selected pin.
+
+  use-unfiltered-output:
+    type: boolean
+    description: |
+      Decide whether to use the unfiltered output.
+
+  enable-output-invert:
+    type: boolean
+    description: |
+      Decide whether to invert the comparator output.
+
+  hysteresis-level:
+    type: int
+    enum:
+      - 0
+      - 1
+      - 2
+      - 3
+    default: 0
```

Comment:
all the default values need justifications in the descriptions 

https://builds.zephyrproject.io/zephyr/pr/71890/docs/build/dts/bindings-upstream.html#rules-for-default-values